Automattic/wp-calypso
# Template Selector: Use patterns API

Fixes #46600

#### Changes proposed in this Pull Request

* Replace the current API endpoint for pulling starter page templates with the patterns API so that new templates can be added easily via the patterns source site. No updates have been made to the existing design.

<img width="1607" alt="Screen Shot 2020-12-17 at 2 27 18 PM" src="https://user-images.githubusercontent.com/1464705/102551228-fb82c480-4073-11eb-984e-e43eaf923a44.png">

#### Testing instructions



* Apply this PR
* Add a new page on your WordPress site
* Confirm that you see templates matching above
* Select and use a template, then in the editor sidebar select "Change Layout".
* Confirm that the selected template is shown under "Current" at the top of the window.
* Confirm you can publish a page with the selected template.
